### PR TITLE
update info in admins-documentation.md and adguard-logs.md

### PR DESCRIPTION
**Pull request by anastasiia-fedotova**

### CODE REVIEW

**Review by el-termikael — APPROVED** [review]

File: docs/adguard-for-windows/admins-documentation.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ If you want to install AdGuard on a Windows 7 computer, make sure that it has .N
 
 :::info
 
-Automatic updates are disabled when you install AdGuard for Windows from MSI. If you want to allow updates for a user (which is not recommended because centralized updates will become impossible), set the value of the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key to `YES` (case insensitive). In this case automatic updates will be allowed, any other value or no value for this parameter disables automatic updates.
+Starting with AdGuard v7.17 for Windows, automatic updates are enabled by default when you install the application from MSI. We recommend that you disable updates for a user so that AdGuard for Windows is updated centrally. To do this, add the ForbidCheckUpdates=Yes key, which is located at the following path: HKEY_LOCAL_MACHINE\SOFTWARE\Adguard.
+
+In older versions of AdGuard for Windows an automatic updates are disabled by default. To allow updates for a user, set the value of the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key to `YES` (case insensitive).
```
Comment (el-termikael) [inline docs/adguard-for-windows/admins-documentation.md]: an automatic updates

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by Danila-Arg — APPROVED** [review]

File: docs/adguard-for-windows/solving-problems/adguard-logs.md [hunk 2]
```diff
@@ -43,7 +43,9 @@ Sometimes support team members may ask you to send *trace* logs. Then you will n
 
 1. Open the console (type `cmd` in the start panel).
 
-1. Run the app with the command `C:\"Program Files (x86)"\Adguard\Adguard.exe /trace` if you are using 64-bit Windows, and `C:\"Program Files"\Adguard\Adguard.exe /trace` in case if you use 32-bit one.
+1. Run the app with the command `C:\"Program Files"\Adguard\Adguard.exe /trace` regardless of Windows bitness if you have AdGuard v7.16 for Windows or later installed.
```
Comment (el-termikael) [inline docs/adguard-for-windows/solving-problems/adguard-logs.md:46]: if you have installed AdGuard v7.16 for Windows or later

File: docs/adguard-for-windows/solving-problems/adguard-logs.md [hunk 3]
```diff
@@ -43,7 +43,9 @@ Sometimes support team members may ask you to send *trace* logs. Then you will n
...
+
+In case you have an older AdGuard for Windows version, run the app with the command `C:\"Program Files (x86)"\Adguard\Adguard.exe /trace` if you are using 64-bit Windows, and `C:\"Program Files"\Adguard\Adguard.exe /trace` in case if you use 32-bit one.
```
Comment (el-termikael) [inline docs/adguard-for-windows/solving-problems/adguard-logs.md:48]: If you have an older version of AdGuard for Windows, run the application with the command `C:\"Program Files (x86)"\Adguard\Adguard.exe /trace` if you are using 64-bit Windows, and `C:\"Program Files"\Adguard\Adguard.exe /trace` if you are using 32-bit.

File: docs/adguard-for-windows/solving-problems/adguard-logs.md (hunk 2 above)
Comment (el-termikael) [reply docs/adguard-for-windows/solving-problems/adguard-logs.md:46]: Run the application

**Review by el-termikael — APPROVED** [review]

**Pull request merged**